fetch/toc-html.es6

```diff
@@ -2,9 +2,10 @@ import 'whatwg-fetch';
 import { parseRawHtml } from '../parse-raw';
 
 const TOC_HTML = 'toc.xhtml';
+const FOLDER_NAME = 'OPS';
 
-export default function containerHtml(uri, source=TOC_HTML) {
-  return fetch(`${uri}/OPS/${source}`).
+export default function containerHtml(uri, source=TOC_HTML, folder=FOLDER_NAME) {
```

Comment:
I'm wondering if folder should be part of uri. What do you think @Lphmedia? Should we keep it separate as we have it now or merge them?